README.md

```diff
@@ -79,30 +79,33 @@ The tests are automatically run by Travis CI on every PR and push to
 the repo.  There are two separate sets of tests: `tests/mypy_test.py`
 runs tests against [mypy](https://github.com/python/mypy/), while
 `tests/pytype_tests.py` runs tests against
-[pytype](https://github.com/google/pytype/).  The script runtests.sh
-runs both sets of tests and flake8 over all .pyi files.
+[pytype](https://github.com/google/pytype/).
 
 Both sets of tests are shallow -- they verify that all stubs can be
 imported but they don't check whether stubs match their implementation
 (in the Python standard library or a third-party package).  Also note
 that each set of tests has a blacklist of modules that are not tested
 at all.  The blacklists also live in the tests directory.
 
-To manually run the mypy tests, you need to have Python 3.5 or higher.
+To manually run the mypy tests, you need to have Python 3.5 or higher;
+Python 3.6.1 or higher is recommended.
 Run:
 ```
-$ python3.5 -m venv .venv3
+$ python3.6 -m venv .venv3
 $ source .venv3/bin/activate
-(.venv3)$ pip install -r requirements-tests-py3.txt
+(.venv3)$ pip3 install -r requirements-tests-py3.txt
 ```
-This will install mypy-lang, typed-ast, and flake8. You can then run
-mypy tests and flake8 tests by invoking:
+This will install mypy (you need the latest master branch from GitHub),
+typed-ast, and flake8. You can then run mypy tests and flake8 tests by
+invoking:
 ```
-(.venv3)$ python tests/mypy_test.py
+(.venv3)$ python3 tests/mypy_test.py
 ...
 (.venv3)$ flake8
 ...
 ```
+(Note that flake8 only works with Python 3.6 or higher.)
```

Comment:
I got the following errors with 3.5:
```
Traceback (most recent call last):
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/plugins/manager.py", line 175, in load_plugin
    self._load(verify_requirements)
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/plugins/manager.py", line 147, in _load
    self._plugin = resolve()
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2361, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/pyi.py", line 182
    can_have_strict_equals: Optional[int] = None
                          ^
SyntaxError: invalid syntax

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/guido/src/typeshed/.venv3/bin/flake8", line 11, in <module>
    sys.exit(main())
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/main/cli.py", line 16, in main
    app.run(argv)
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/main/application.py", line 328, in run
    self._run(argv)
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/main/application.py", line 315, in _run
    self.initialize(argv)
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/main/application.py", line 297, in initialize
    self.find_plugins()
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/main/application.py", line 147, in find_plugins
    self.check_plugins.load_plugins()
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/plugins/manager.py", line 392, in load_plugins
    plugins = list(self.manager.map(load_plugin))
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/plugins/manager.py", line 284, in map
    yield func(self.plugins[name], *args, **kwargs)
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/plugins/manager.py", line 390, in load_plugin
    return plugin.load_plugin()
  File "/Users/guido/src/typeshed/.venv3/lib/python3.5/site-packages/flake8/plugins/manager.py", line 183, in load_plugin
    raise failed_to_load
flake8.exceptions.FailedToLoadPlugin: Flake8 failed to load plugin "Y0" due to invalid syntax (pyi.py, line 182).
```
I think you once realized this -- typeshed's .travis.yml runs flake8 (and only flake8) under Python 3.6.